style.css

```diff
@@ -257,7 +265,7 @@ hr {
 
 input[type=text], select, textarea{
 	width: 100%;
-	height: 60px;
```

Comment:
I improved the sizing of the input fields. Now it looks much better and less blocky.